server/testdriver/core/pom.xml

```diff
@@ -73,5 +73,18 @@
          <scope>compile</scope>
       </dependency>
    </dependencies>
+
+   <build>
+      <plugins>
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+               <release>11</release>
```

Comment:
why is this needed ?

Reply:
Without that I can't use `process.pid`, I'm explicitly setting JDK 11 as compiler

Reply:
Forked mode shouldn't support kill, stop or cont. Please remove `process.pid`. If you wish to `stop` the server. Call the stop endpoint from the `RestClient`

Reply:
Ah, you need to have the PID of the process

Reply:
> Forked mode shouldn't support kill, stop or cont. Please remove `process.pid`. If you wish to `stop` the server. Call the stop endpoint from the `RestClient`

@diegolovison I'm stopping the server with RestClient already, you can see here https://github.com/infinispan/infinispan/blob/704ef6d1b727a7ca79a462290fed703879467b5c/server/testdriver/core/src/main/java/org/infinispan/server/test/core/ForkedInfinispanServerDriver.java#L69

Reply:
We still support Java 8 to run the tests.
What will be the behavior when we enable the profile `java8-test` when running the tests?

If we are going to support `kill -9` why can't we support `kill -STOP` or `kill -CONT`? IMHO, I do prefer have those kinds of tests with `ContainerInfinispanServerDriver`

Reply:
> Forked mode shouldn't support kill, stop or cont. Please remove `process.pid`. If you wish to `stop` the server. Call the stop endpoint from the `RestClient`

I disagree with this. I see no point in hobbling `FORKED` mode. It should have the same capabilities as the other modes of `InfinispanServerRule`.

BTW, ping @rhusar you'll be interested in this PR

Reply:
@alanfx @gustavolira Yes - very interested - thanks for the ping. Will take it for a spin later this week.

Reply:
@diegolovison 

>Forked mode shouldn't support kill

I am not following why, could you explain? Actually, FORKED mode is kind of ideal for supporting a JVM kill.

Reply:
@rhusar, when you start multiple processes on the same OS and then start doing some chaos testing like ( kill, stop or cont ) those tests will start failing sometimes and you won't be able to reproduce it locally ( flaky tests ). The best option is to use Containers for those kinds of tests or isolates in different VMs/Machines.

This is why for me it is a `-1` to support kill, stop or cont.